batch_seqr_loader/seqr_load.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/env python3
+
+"""
+Hail script to submit on a dataproc cluster. Converts input multi-sample VCFs
+into a matrix table, which annotates and prepares to load into ES.
+"""
+
+import logging
+from typing import Optional, List
+
+import click
+import hail as hl
+
+from luigi_pipeline.lib.model.seqr_mt_schema import SeqrVariantsAndGenotypesSchema
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+
+logger = logging.getLogger()
+
+
+@click.command()
+@click.option(
+    '--source-path',
+    'source_paths',
+    multiple=True,
+    required=True,
+)
+@click.option(
+    '--dest-mt-path',
+    'dest_path',
+    required=True,
+)
+@click.option(
+    '--bucket',
+    'bucket',
+    required=True,
+)
+@click.option('--genome-version', 'genome_version', default='GRCh38')
+@click.option(
+    '--reference-ht',
+    'reference_path',
+    required=True,
+    help='Path to the Hail table storing the reference variants.',
+    default='gs://seqr-reference-data/GRCh38/all_reference_data/v2/combined_reference_data_grch38-2.0.2.ht',
```

Comment:
```suggestion
    default='gs://cpg-seqr-reference-data/GRCh38/all_reference_data/v2/combined_reference_data_grch38-2.0.3.ht',
```

I've copied these references to the Australian bucket to avoid egress for Broad (and should be quicker for us). I've only copied the 2.0.3 all_reference_data ht though.

Reply:
Hmm, getting error accessing it: https://batch.hail.populationgenomics.org.au/batches/2557/jobs/3

```
hail.utils.java.FatalError: GoogleJsonResponseException: 403 Forbidden
{
  "code" : 403,
  "errors" : [ {
    "domain" : "global",
    "message" : "seqr-test-709@hail-295901.iam.gserviceaccount.com does not have storage.objects.get access to the Google Cloud Storage object.",
    "reason" : "forbidden"
  } ],
  "message" : "seqr-test-709@hail-295901.iam.gserviceaccount.com does not have storage.objects.get access to the Google Cloud Storage object."
}

Java stack trace:
java.io.IOException: Error accessing gs://cpg-seqr-reference-data/GRCh38/all_reference_data/v2/combined_reference_data_grch38-2.0.3.ht
```